src/layout-manager/TabbedDashboardView.vue

```diff
@@ -41,7 +41,7 @@
     @up="goUpOneFolder()"
   )
 
-  //- p.load-error(v-show="loadErrorMessage" @click="authorizeAfterError"): b {{ loadErrorMessage }}
+  p.load-error(v-show="loadErrorMessage" @click="authorizeAfterError"): b {{ loadErrorMessage }}
```

Comment:
I need to look into this `authoizeAfterError()` call that you re-enabled, I think it is a mistake